README.md

```diff
@@ -1,6 +1,10 @@
-# ukbot
+<h1 align="center">UKBot</h1>
 
-Bot for updating results in writing contests at Wikipedia, deployed at [ukbot.wmflabs.org](https//ukbot.wmflabs.org).
+<p align="center">Bot for updating results in writing contests at Wikipedia, deployed at <a href="https://ukbot.toolforge.org/">ukbot.toolforge.org</a>.</p>
+
+<div align="center">
+  <p>Create with ❤︎ by <a href="https://github.com/WikimediaNorge/UKBot/graphs/contributors">contributors</a></p>
```

Comment:
\* Created

And I think Dan Michael should be mentioned by name here, without him there wouldn't be anything at all